### ISSUE TITLE
v-for with two params doesn't work

### ISSUE THREAD

**Issue by dpetrovaliev** [inxilpro/IntellijAlpine#27]
Looks like v-for with two params case isn't handled.
Plugin version: v0.4.2

Screenshot:
<img width="868" alt="Screenshot 2021-08-30 at 17 53 07" src="https://user-images.githubusercontent.com/53557596/131358929-acfea755-9cfa-430a-8cf2-86e2ca4f8f62.png">

**Comment by inxilpro** [comment]
Yeah. This is a known issue. I'm not sure when a fix will be possible—right now I'm essentially using the IntelliJ javascript parser for Alpine directives, but because `x-for` implements custom syntax that's not valid javascript. The only solution would be to implement a custom parser, which is a pretty big thing to take on :/
